src/main/java/com/mindee/documentparser/DocumentManipulation.java

```diff
@@ -0,0 +1,21 @@
+package com.mindee.documentparser;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class DocumentManipulation {
+
+  public enum DocumentManipulationMode{
```

Comment:
``PageOptionsOperation`` to be consistent with Python SDK, Node.js SDK and .NET SDK

Reply:
willr refactor